### PR TITLE
add alphabetical type

### PR DESCRIPTION
**Pull request by this-mojtaba**
adding alphabetical type in to the project

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "editor.formatOnType": true,
+        "editor.insertSpaces": true,
+        "editor.tabSize": 2
+    },
+    "[typescriptreact]": {
+        "editor.formatOnType": true,
+        "editor.insertSpaces": true,
+        "editor.tabSize": 2
+    }
+}
```
Comment (disizali) [inline .vscode/settings.json]: could you remove this file, please?

Comment (this-mojtaba) [reply .vscode/settings.json]: of course.

File: examples/interactive.ts [hunk 2]
```diff
@@ -15,6 +15,13 @@ const results = await interaction.interact([
     type: InteractType.numeric,
     bold: false,
   },
+  {
+    name: "alpha",
+    message: "enter a key",
```
Comment (disizali) [inline examples/interactive.ts]: Replace this example with a more realistic one.
like: `type your favorite color`

File: src/types/alphabetical.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  public isAlphabetical(input: string): boolean {
+    const enAlphabet = input.charCodeAt(0) >= 65 && input.charCodeAt(0) <= 122;
+    // TODO: Adding persian, greek alphabet
+
+    return enAlphabet;
+  }
+
+  protected async getUserChar(): Promise<string> {
```
Comment (disizali) [inline src/types/alphabetical.ts]: this function name must be plural.

**Review by disizali — COMMENTED** [review]
thank you @Mojtaba-79, glad to see your cooperation.
please do `deno fmt` to make ci green and two nitpicks ...

File: src/types/alphabetical.ts (hunk 3 above)
Comment (this-mojtaba) [reply src/types/alphabetical.ts]: thank you for your guidance dear @disizali

**Review by disizali — APPROVED** [review]
LGTM, thank you again!

**Pull request merged**